runtime/service/src/main/java/org/apache/polaris/service/events/CatalogPolicyServiceEvents.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.polaris.service.events;
+
+import org.apache.polaris.service.types.AttachPolicyRequest;
+import org.apache.polaris.service.types.CreatePolicyRequest;
+import org.apache.polaris.service.types.DetachPolicyRequest;
+import org.apache.polaris.service.types.GetApplicablePoliciesResponse;
+import org.apache.polaris.service.types.LoadPolicyResponse;
+import org.apache.polaris.service.types.UpdatePolicyRequest;
+
+/**
+ * Event records for Catalog Policy operations. Each operation has corresponding "Before" and
+ * "After" event records.
+ */
+public class CatalogPolicyServiceEvents {
+
+  // Policy CRUD Events
+  public record BeforeCreatePolicyEvent(
```

Comment:
Sorry to insist on naming, but it's still inconsistent.

In #2482 we have `(Before|After)(Object)(Verb)Event`.

Here we have `(Before|After)(Verb)(Object)Event`.

(I find verb-object more natural than object-verb btw, but I'll defer the decision to native English speakers.)